clients/ms_setting.h

```diff
@@ -148,6 +148,11 @@ typedef struct setting
   int mult_key_num;                     /* number of keys used by multi-get once */
   size_t win_size;                      /* item window size per connection */
   bool udp;                             /* whether or not use UDP */
+  bool tls;                             /* whether or not use TLS */
+  bool tls_cert;                        /* TLS certificate file */
+  bool tls_key;                         /* TLS private key file */
+  bool tls_ca;                          /* TLS CA file */
+  bool tls_skip_verify;                 /* whether or not to skip TLS peer verification*/
```

Comment:
specifically this is the hostname verification, right?

Reply:
see above answer

Reply:
Again, rename to `tls_skip_hostname_verify` to avoid confusion.